docker/dev/Dockerfile.torch-disc

```diff
@@ -8,5 +8,5 @@ RUN apt-get update -y && \
     rm /usr/bin/python && \
     ln -s /usr/bin/python3.8 /usr/bin/python && \
     python -m pip install --upgrade pip && \
-    python -m pip install cpython pyyaml typing_extensions virtualenv numpy
-RUN python -m pip install https://bladedisc-ci.oss-cn-hongkong.aliyuncs.com/download/torch-ltc/torch-1.12.0a0%2Bgitd9896b8-cp38-cp38-linux_x86_64.whl
\ No newline at end of file
+    python -m pip install cpython pyyaml typing_extensions virtualenv numpy torchvision
+RUN python -m pip uninstall pytorch && python -m pip install https://bladedisc-ci.oss-cn-hongkong.aliyuncs.com/download/torch-ltc/torch-1.12.0a0%2Bgitd9896b8-cp38-cp38-linux_x86_64.whl
```

Comment:
what magic is in this torch wheel?

Reply:
> we probably should build torchvision with pinned git sha as well

Agree with that, maybe we can do that in the future PR because we just use the Mnist dataset preprocessing, and I will leave an issue to trace it.

Reply:
> what magic is in this torch wheel?

build from the [lazy_tensor_staging branch of PyTorch](https://github.com/pytorch/pytorch/tree/lazy_tensor_staging).